docs/UserGuide.adoc

```diff
@@ -233,10 +233,30 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
-// tag::dataencryption[]
-=== Encrypting data files `[coming in v2.0]`
+=== Data visualization
```

Comment:
Can we keep the format consistent when documenting new commands in the User Guide? Look at the other commands given above, for example, " === Exiting the program : `exit`"